test/functional/admin/bulk_republishing_controller_test.rb

```diff
@@ -205,7 +205,7 @@ class Admin::BulkRepublishingControllerTest < ActionController::TestCase
     assert_response :forbidden
   end
 
-  test "GDS Admin users should be able to POST :republish_documents_by_organisation with an existing organisation slug, creating a RepublishingEvent for the current user" do
+  test "GDS Admin users should be able to POST :republish_documents_by_organisation with an existing organisation slug and a reason, creating a RepublishingEvent for the current user" do
```

Comment:
For next time, I think putting things like this (and some other refactorings) into a separate PR would help reduce some of the reviewing burden, as there's a lot of change in this PR!

Reply:
Yup - I normally pull them out at the end, but rushed this one a bit!